### ISSUE TITLE
Locked notes can be deleted but not restored

### ISSUE THREAD

**Issue by kittey** [r888888888/danbooru#2545]
On a post with locked notes it's still possible to delete notes. This should obviously not to be possible.

While the post is note-locked, deleted notes cannot be restored from the note history. Clicking "Revert to" on a note revision pops up the confirmation dialogue, but when "OK" is clicked, only the dialogue box vanishes and nothing else happens. This should show some kind of error message.

In other words, a note-locked post can still be vandalized by anyone, but only Builder+ can fix it.

**Comment by ToksT** [comment]
already fixed
